src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/utils/UpdateSessionTemplateValidator.kt

```diff
@@ -2,17 +2,25 @@ package uk.gov.justice.digital.hmpps.visitscheduler.utils
 
 import org.springframework.beans.factory.annotation.Value
```

Comment:
Should this be in the uk.gov.justice.digital.hmpps.visitscheduler.validators package

Reply:
Updated.